phoebe/parameters/dataset.py

```diff
@@ -778,8 +777,86 @@ def mesh(syn=False, as_ps=True, **kwargs):
 
     return ParameterSet(params) if as_ps else params, constraints
 
+def vis(syn=False, as_ps=True, **kwargs):
+    """
+    Create a <phoebe.parameters.ParameterSet> for an interferometric visibility |V|^2 dataset.
+
+    Note: See lc().
+
```

Comment:
these will need full docstrings since they are used to generate the API docs on the website and all new qualifiers will need to be added to `forbidden_labels` (defined in `parameters.py`, looks like you already added at least some or maybe even all of them) and these methods included in `test_forbidden_parameters`.  This prevents the user from naming a dataset something that conflicts with a qualifier and would result in ambiguous filtering when using twigs.

Reply:
Corrected. Copy & paste.